src/lib/controllers/app-controller.js

```diff
@@ -59,10 +88,9 @@ export default class AppController {
         [enGB, Locales.en_GB]
       ])
     }
-    this.l10n = new L10n()
-    config.messageBundles.forEach(mb => this.l10n.addMessageBundle(mb))
-    this.l10n.setLocale(config.defaultLocale)
-
-    Vue.prototype.$l10n = this.l10n
+    const l10n = new L10n()
+    config.messageBundles.forEach(mb => l10n.addMessageBundle(mb))
+    l10n.setLocale(config.defaultLocale)

```

Comment:
I think that it could be easily updated by expanding config from only appId to more including locale.
I suggest to do it later, when we would define properties for the app